src/libs/TransactionUtils.ts

```diff
@@ -307,11 +308,30 @@ function getOriginalAmount(transaction: Transaction): number {
     return Math.abs(amount);
 }
 
+/**
+ * Verify if the transaction is of Distance request and is not fully ready:
+ * - it has a zero amount, which means the request was created offline and expects the distance calculation from the server
+ * - it is in `isLoading` state, which means the waypoints were updated offline and the distance requires re-calculation
+ */
+function isLoadingDistanceRequest(transaction: OnyxEntry<Transaction>): boolean {
```

Comment:
Something about the name of this function is throwing me off. I think it's because it assumes something about how the rest of the application is working. This makes it confusing when seeing it inside of `getMerchant()` because the first thing I wonder is "Why does isLoading have anything to do with the merchant that's returned?"

The suggestion I have is to rename this method to `isDistanceBeingCalculated` or `hasServerCalculatedDistance`. 

Thoughts?

Reply:
I like the `isDistanceBeingCalculated` name, will rename, thanks